streamly-bytestring.cabal

```diff
@@ -31,8 +31,11 @@ library
   ghc-options: -Wall -O2
   build-depends:
       base >=4.7 && <5
-    , bytestring >=0.10.0 && <=0.11.3.1
-    , streamly == 0.8.1.* || == 0.8.2.* || == 0.8.3.*
+    , bytestring    == 0.11.0.*
+                 || == 0.11.1.*
+                 || == 0.11.2.*
+                 || ( >= 0.11.3.1 && < 0.11.4 )
```

Comment:
0.11.4.0 is not supported? That's the latest version.
Is there is problem with 0.11.3.0? That version is excluded.